IntegrationTests/UI.IntegrationTests/UserControls/GitBlameParserTest.cs

```diff
@@ -14,7 +14,6 @@ public void GetOriginalLineInPreviousCommit()
         // line content is: internal bool ExecuteCommand(Command cmd)
 
         string gitExtensionsRepoPath = Path.GetFullPath(@"..\..\..\..\..\..\..");
-        Assert.IsTrue(gitExtensionsRepoPath.EndsWith("gitextensions"));
```

Comment:
Check for existence of e.g. `GitExtensions.sln` **in** `gitExtensionsRepoPath`?

Reply:
That would replace this check, but I do not see that as adding any value.

Reply:
It would clearly indicate if the test is run from an unexpected folder. But there is no urgent need for this check. So, as you prefer.

Reply:
Is there a reason we can't leverage `TestContext`?
![image](https://github.com/gitextensions/gitextensions/assets/4403806/2b1ea59a-3192-4eb1-9f8d-49baa586778d)